client/src/App.js

```diff
@@ -16,6 +17,9 @@ function App() {
         <Route path="/about">
           <DoctorAbout />
         </Route>
+        <Route path="/form">
```

Comment:
Differentiate the path from patients' form, maybe use path `/doctor/form` or `/doctor-form` to . Same for `/about` path

Reply:
I think we should use conditional rendering for that. Show pages according to the type of account the user is using. If it is a doctor use doctor's page, else patient's page.